tools/llm_bench/task/image_generation.py

```diff
@@ -95,13 +95,85 @@ def run_image_generation(image_param, num, image_id, pipe, args, iter_data_list,
     stable_diffusion_hook.clear_statistics()
 
 
+def run_image_generation_genai(image_param, num, image_id, pipe, args, iter_data_list, proc_id, mem_consumption, tokenizer):
+    set_seed(args['seed'])
+    input_text = image_param['prompt']
+    image_width = image_param.get('width', DEFAULT_IMAGE_WIDTH)
+    image_height = image_param.get('height', DEFAULT_IMAGE_HEIGHT)
+    nsteps = image_param.get('steps', DEFAULT_INFERENCE_STEPS if 'lcm' not in args["model_name"] else LCM_DEFAULT_INFERENCE_STEPS)
+    guidance_scale = image_param.get('guidance_scale', None)
+    log.info(
+        f"[{'warm-up' if num == 0 else num}][P{image_id}] Input params: Batch_size={args['batch_size']}, "
+        f'steps={nsteps}, width={image_width}, height={image_height}, guidance_scale={guidance_scale}'
+    )
+    result_md5_list = []
+    max_rss_mem_consumption = ''
+    max_uss_mem_consumption = ''
+    max_shared_mem_consumption = ''
+    if (args['mem_consumption'] == 1 and num == 0) or args['mem_consumption'] == 2:
+        mem_consumption.start_collect_memory_consumption()
+    additional_args = {}
+    if guidance_scale is not None:
+        additional_args["guidance_scale"] = guidance_scale
+    else:
+        if 'lcm-sdxl' in args['model_type']:
```

Comment:
why only sdxl?

Reply:
I also belive that these parameters like guidance_scale, num steps e.g. are common for both inference pipelines, can we move that into utils and reuse for both?